meet/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG docker_repo=zokradonh
+FROM ${docker_repo}/kopano_base
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG ADDITIONAL_KOPANO_PACKAGES=""
+ARG DOWNLOAD_COMMUNITY_PACKAGES=1
+ARG KOPANO_MEET_REPOSITORY_URL="file:/kopano/repo/meet"
+ARG KOPANO_MEET_VERSION=newest
+ARG KOPANO_REPOSITORY_FLAGS="trusted=yes"
+ARG RELEASE_KEY_DOWNLOAD=0
+
+# install Kopano Core and refresh ca-certificates
+RUN \
+    # community download and package as apt source repository
+    . /kopano/helper/create-kopano-repo.sh && \
+    if [ ${DOWNLOAD_COMMUNITY_PACKAGES} -eq 1 ]; then \
+        dl_and_package_community "meet"; \
+    fi; \
+    echo "deb [${KOPANO_REPOSITORY_FLAGS}] ${KOPANO_MEET_REPOSITORY_URL} ./" > /etc/apt/sources.list.d/kopano.list; \
+    # save kopano version
+    echo "meet-${KOPANO_MEET_VERSION}" > /kopano/buildversion; \
+    # install apt key if supported kopano
+    if [ ${RELEASE_KEY_DOWNLOAD} -eq 1 ]; then \
+        curl -s -S -o - "${KOPANO_MEET_REPOSITORY_URL}/Release.key" | apt-key add -; \
+    fi; \
+    # install
+    apt-get update && \
+    set -x && \
+    apt-get install --no-install-recommends -y \
+        kopano-kwebd \
```

Comment:
This is a nice idea - we probably should add a dedicated mode to kweb to serve an individual (single) pwa.

Reply:
yes, here I also was a bit indifferent. Should it be one docker container/image per pwa or just one including all (future) pwas. I think this can definitely change once there are more apps.